pkg/operator/status.go

```diff
@@ -185,6 +185,18 @@ func (optr *Operator) getOrCreateClusterOperator() (*osconfigv1.ClusterOperator,
 			Resource: "namespaces",
 			Name:     optr.namespace,
 		},
+		{
+			Group:     "machine.openshift.io",
+			Resource:  "machines",
+			Name:      "",
+			Namespace: optr.namespace,
+		},
+		{
+			Group:     "machine.openshift.io",
+			Resource:  "machinesets",
+			Name:      "",
+			Namespace: optr.namespace,
+		},
```

Comment:
If you end up having to rebase for some reason, could you add the `baremetalhosts` resource from group `metal3.io`, as well?

Reply:
@russellb we don't need to rebase (pull new code under vendor) at all, right? Just open another PR and extend the list with:
```
{
	Group:     "metal3.io",
	Resource:  "baremetalhosts",
	Name:      "",
	Namespace: optr.namespace,
},
```